filebeat/module/nats/log/_meta/fields.yml

```diff
@@ -0,0 +1,17 @@
+- name: log
+  type: group
+  description: >
+    Nats log files
+  fields:
+    - name: pid
```

Comment:
As the Nats module is only in 7.0, we should not add these migration aliases. This is only for modules from 6.x.

Reply:
Hi @ruflin. Documentation of these fields has confused us and we want your help.

As you have noticed, all our extracted fields (for this first PR) are not Nats specific but they belong to the common schema.

They question is how they should be documented in 7? Because we used this pattern after looking at redis module which does the same.

Do we need to leave key "fields" empty or should we document them somehow? Can you provide an example of let's say process.pid?

Reply:
You can leave fields.yml mostly empty but it will need the `release: beat` docs inside. 

The fields are already document but I understand the concern, that they are not directly documented with the module / fileset. We don't have here a solution yet.